### ISSUE TITLE
I get the error (0 , $csb__utils.fromEntries) is not a function for a sandbox that works when exported

### ISSUE THREAD

**Issue by AndrewPrifer** [codesandbox/codesandbox-client#5100]
# 🐛 bug report

## Preflight Checklist

- [x] I have read the
      [Contributing Guidelines](https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md)
      for this project.
- [x] I agree to follow the
      [Code of Conduct](https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md)
      that this project adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem
I get the following error: `(0 , $csb__utils.fromEntries) is not a function`. If I export the codesandbox, it works, so the code is not the issue. The problem is related to the [react-three-editable](https://github.com/AndrewPrifer/react-three-editable) package. 3.0.0 of react-three-editable works, while 0.3.1 breaks csb. The strange thing is that there are trivial changes in the package [from 0.3.0 to 0.3.1](https://github.com/AndrewPrifer/react-three-editable/compare/v0.3.0...v0.3.1).

## How has this issue affected you? What are you trying to accomplish?
Trying to set up an example for a package I'm making.

### To Reproduce

Install react-three-editable 0.3.1.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |  50accd694
| Browser          | Chrome 86.0.4240.183
| Operating System | MAcOS 10.15.7

**Comment by SaraVieira** [comment]
Hey!

Seems the issue is actually the small package you added "@chakra-ui/theme-tools" is not being transpiled and the code breaks, installing it fixes the issue: 

https://codesandbox.io/s/minimal-r3e-demo-forked-gxkv0?file=/src/App.tsx

Why that is not getting transpiled I am not quite sure :/

Seems like something on their side will ping @CompuIves so he can take a look.

Also that is an amazing package!!!

**Comment by AndrewPrifer** [comment]
Thanks so much for taking the time to look into this! Also thank you for the encouraging words :)

**Comment by CompuIves** [comment]
Heyo! We found what the cause of the rather cryptic error was!

It seems like this version of `@chakra-ui/theme-tools` requires `"@chakra-ui/utils": "^1.0.0-next.3"`, which it also imports `fromEntries` from. However, version `1.0.0-next.3` doesn't export `fromEntries`, it's actually exported from version `1.0.0-next.4` onwards. Unfortunately we install the exact version in this case (`1.0.0-next.3`), which causes the error. It works locally, because Yarn installs a later version as optimization instead.

We found a fix for it to be to update `react-three-editable`, that one contains a newer version of the `@chakra` suite and I believe the issue is fixed in those versions. Hopefully this explanation helps understanding what the issue is!
